src/main/java/melky/resourcepacks/SpriteOverride.java

```diff
@@ -136,6 +136,11 @@
 	MINIMAP_ORB_WORLD_MAP_PLANET(SpriteID.MINIMAP_ORB_WORLD_MAP_PLANET, Folder.OTHER),
 	MINIMAP_ORB_WORLD_MAP_PLANET_HOVERED(SpriteID.MINIMAP_ORB_WORLD_MAP_PLANET_HOVERED, Folder.OTHER),
 
+	DIALOG_BACKGROUND(SpriteID.DIALOG_BACKGROUND, Folder.OTHER),
+	BANK_TAB(SpriteID.BANK_TAB, Folder.OTHER),
+	BANK_TAB_HOVERED(SpriteID.BANK_TAB_HOVERED, Folder.OTHER),
+	BANK_TAB_SELECTED(SpriteID.BANK_TAB_SELECTED, Folder.OTHER),
```

Comment:
I'd rather have these in a new BANK folder, in there is more bank stuff to add and not to spam the other folder